### DISCUSSION TITLE
Can we run a eslint --fix?

### DISCUSSION THREAD

**Discussion by 511234** [Pjaijai/Referalah#123] [討論 | General]
it'd be easier to do code review, we wouldn't have to see format changes in a feature commit.

**Comment by Pjaijai** [comment]
Sure, Will create PR later. I am thinking installing prettier-plugin-tailwindcss. that auto sorting our tailwind className on Save

**Comment by 511234** [comment]
sure, and also [husky](https://typicode.github.io/husky/) to lint before git push maybe?

**Reply by Pjaijai** [reply]
Awesome. Thats what I want to do as well. Haven't add husky.
